src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export function Person({ person }) {}
+import React from 'react';
+
+export const Person = ({ person:
+  { name, age, sex, isMarried, partnerName } }) => (
+
+    <section className="Person">
+      {name && (
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      )}
+      {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+      )}
```

Comment:
```suggestion
      <p className="Person__age">{age && `I am ${age}`}</p>
```

Reply:
Thanks for a review, but why this solution is better ?
In my opinion, we have to remove the whole <p> tag if no age passed.